src/components/media_manager/include/media_manager/streamer_adapter.h

```diff
@@ -87,7 +87,7 @@ class StreamerAdapter : public MediaAdapterImpl {
   };
 
  private:
-  std::atomic_int current_application_;
+  int32_t current_application_;
```

Comment:
Any reason why `std::atomic` is no longer needed for this variable?

Reply:
Done in f0de706